app/decorators/course_decorator.rb

```diff
@@ -13,9 +13,9 @@ def vacancies
 
   def on_find(provider = object.provider)
     if object.findable?
-      if current_cycle?
+      if current_cycle_and_open?
         h.govuk_link_to("Yes - view online", h.search_ui_course_page_url(provider_code: provider.provider_code, course_code: object.course_code))
-      elsif next_cycle?
+      else
         "Yes – from October"
```

Comment:
do we need to give 1 or 8 or just nothing?

Reply:
![image](https://user-images.githubusercontent.com/470137/65773016-607e9180-e133-11e9-98d4-0d71af723191.png)

Reply:
from October is fine I think.